worlds/mario_kart_double_dash/__init__.py

```diff
@@ -279,7 +289,7 @@ def add_client_to_launcher() -> None:
                 c.func = launch_client
                 return
     if not found:
-        components.append(Component("Mario Kart Double Dash Client", "MKDDClient", func=launch_client))
+        components.append(Component("Mario Kart Double Dash Client", func=launch_client))
```

Comment:
Is there a reason for this change?
This seems to be working when started via launcher ui, but when I try to start the launcher with "MKDDClient" argument, it won't launch the client.
```Launcher.py: error: bad url, found MKDDClient, expected url in form of archipelago://archipelago.gg:38281```

Reply:
Unfortunately, this breaks as soon as the world is bundled with the archipelago building process using cx_freeze.
The Component class second attribute is "script_name", which whenever set is used by cx_freeze to find a dedicated client.py in the root folder of ap - when it's not there, the world gets excluded.
The compoent class states:
_script_name: Recommended to use func instead; Name of file to run when the component is called_

 If you still want to be able to call your client via CLI, I am not sure if there is an alternative to that though :/

Reply:
It seems that calling with "Mario Kart Double Dash Client" achieves the same thing, so no problems after all 👌